### ISSUE TITLE
code block unstable

### ISSUE THREAD

**Issue by scu2013** [giuspen/cherrytree#213]
my operating system is 64-bit windows 7 and the version of cherry version is the latest version. I use code block in a node, and when I narrowing windows an error occur. like this: 
> This application has request the Runtime to terminate it in unusual way.
more detail:

> I narrow window by drag but not by the button on the upper right corner.

**Comment by giuspen** [comment]
Hi, you mean that you narrow the cherrytree main window with one node with a codebox is selected?

**Comment by scu2013** [comment]
Yes, with one node with a codebox is selected, I narrow the main window and it's down. I just writing some code in a code box and I wanna check something else. So I narrow the main windows but not close it and the error occur. Do I explain myself?

**Comment by giuspen** [comment]
I was not able to reproduce the issue, is it systematic? Could you record your desktop whike reproducing the crash?

**Comment by scu2013** [comment]
before operation, like this:
![Uploading before.png…]()
and after operation, like this:
![Uploading after.png…]()

**Comment by scu2013** [comment]
can you see the figures? btw, should there be another chance when I use wrong password?

**Comment by giuspen** [comment]
I cannot see the pictures. About having another chance if you insert the wrong password you are right, let me see if I can fix that for the next upcoming version

**Comment by scu2013** [comment]
I upload pictures again with picture about password.
![before-wrong-password](https://cloud.githubusercontent.com/assets/8737487/21502394/67bdc2c4-cc8a-11e6-8266-c19483554843.png)
![check-password](https://cloud.githubusercontent.com/assets/8737487/21502390/65b21d72-cc8a-11e6-8cd8-bbee239a2e56.png)
![code-box](https://cloud.githubusercontent.com/assets/8737487/21502397/6b3a782a-cc8a-11e6-96cd-100d09000e81.png)

Are these pictures OK and clear? btw, could I expected markdown support?

**Comment by giuspen** [comment]
I fixed the problem of not asking for the password again in case of error but was not able to reproduce the crash while resizing a window with a codebox. Is this happening also if you create a new document, add one node, add one codebox and resize?

**Comment by milantares** [comment]
Hi,
I would like to add to this thread my experience.
Running Fedora 26, x86_64.

Since using CherryTree for the past month, on multiple occasions the program crashed when I was writing inside the code block at a point when I reached the end of the default size and new line got inserted and the block resized. It doesn't happen every time but with fair amount of regularity.

The code block has always been in SQL in some type of a subnode, should that be relevant.
Thanks
Milan

**Comment by giuspen** [comment]
Thanks Milan, I will disable by default the auto resize of codeboxes to prevent this, you can disable in the preferences. I won't try to fix this before the port to c++/gtk3
